docs/basic-features/eslint.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Next.js uses ESLint to make it easier to find and resolve issues that affect user or
+developer experience.
+---
+
+# ESLint
+
+Next.js uses [ESLint](https://eslint.org/) to make it easier to find and resolve issues that affect
+user or developer experience. A minimal set of Next.js rules are provided by default, but can be
+extended by adding an `.eslintrc` file to your project.
+
+## Default configuration
+
+By default, Next.js provides a set of [recommended ESLint
+rules](https://github.com/vercel/next.js/blob/canary/packages/eslint-plugin-next/lib/index.js#L10-L18)
+that are automatically linted against as part of `next build`. If you would like to control which
+ESLint rules are checked during builds, you will need to add an `.eslintrc` file to the root of your
+project.
+
+Here's an example of an `.eslintrc.json` file:
+
+```json
+{
+  "extends": ["plugin:@next/next/recommended"],
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true,
+      "modules": true
+    }
+  }
+}
+```
+
+- Extending the original base of rules (`plugin:@next/next/recommended`) is highly recommended to
+  catch and fix significant Next.js issues in your application
+- Specify `parserOptions` in the same way as the above example to enable appropriate language
+  features
+
+> If you add an `.eslintrc` file to your application and not include `plugin:@next/next/recommended`
+> to the config, its rules will not be checked during development or production builds. This is
+> **not recommended**.
```

Comment:
Should it throw an error if they do this?

Reply:
A warning is displayed if they decide not to include the plugin. It may be too strict to throw an error and not let users easily opt-out of not including it.

I'll be working on better surfacing of warnings/errors next so I'll evaluate different options to inform the user if they don't include the plugin themselves (error overlay, etc...)  :)